debian/control

```diff
@@ -19,6 +20,7 @@ Build-Depends:
  libpng-dev,
  libwebp-dev,
  pkg-config,
+ xdg-utils,
```

Comment:
This is for the test only, right? I think we should install xvfb for this test to pass in the builders. Did you test this in github?

Reply:
I'm also ok not running this test in the Debian package build.

Reply:
No no, this is for xdg-mime, which apparently gets executed I think in cmake install... The pipeline fails without this either way (and works with this line ;))